docs/quickstart.rst

```diff
@@ -60,7 +60,7 @@ possible using SQLAlchemy:
                                 backref=db.backref('users', lazy='dynamic'))
 
     # Setup Flask-Security
-    user_datastore = SQLAlchemyUserDatastore(db, User, Role)
+    user_datastore = SQLAlchemyUserDatastore(db.session, User, Role)
```

Comment:
I would probably add a comment here that says something like `if using flask-sqlalchemy, use db instead of db.session`